sdk/purview/azure-purview-sharing/CHANGELOG.md

```diff
@@ -1,18 +1,6 @@
 # Release History
 
-## 1.0.0b3 (2023-07-14)
-
-### Bugs Fixed
-
-- Updated broken link in samples README to point to Azure/azure-python-sdk instead of developer fork
-
-### Other Changes
-
-- Migrated test recordings to the azure-sdk-assets repo
-- Added more sanitizers to conftest.py
-
-
-## 1.0.0b3 (2023-06-29)
+## 1.0.0b3 (2023-07-19)
```

Comment:
Did this already release or does the date need to be updated to today?

Reply:
It's already been released. Should the date stay the same for this kind of update?

Reply:
Ah I see. Yes, let's keep it as-is then. The date should reflect the day it was released.